src/components/Accordion/Accordion.tsx

```diff
@@ -92,7 +92,7 @@ class Accordion extends AutoControlledComponent<Extendable<IAccordionProps>, any
     /** Custom styles to be applied for component. */
     styles: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
 
-    /** Custom variables to be applied for component. */
+    /** Override for theme variables to allow modifications of component styling via themes. */
```

Comment:
would suggest `Override for theme site variables.. `, just to be consistent with current naming used.